annotations/static/annotations/js/annotators/appellationlist.js

```diff
@@ -119,8 +154,44 @@ var AppellationListItem = {
         },
         getFormattedDate: function (isodate) {
             return moment(isodate).format('dddd LL [at] LT');
+        },
+        deleteAppellation: function() {
+            this.deleteError = null;
+            Appellation.delete({id: this.appellation.id}).then(response => {
+                // Emit event to parent to remove from list and text display
+                this.$emit('removeappellation', this.appellation);
+            }).catch(error => {
+                if (error.status === 400) {
+                    this.deleteError = "This annotation is used in a relation and cannot be deleted.";
+                } else {
+                    this.deleteError = "Error deleting annotation, Please try again later.";
+                }
+                
+                // Clear error message after 5 seconds
```

Comment:
Error messages should not be automatically cleared. What if the user turns around or goes to get coffee and misses the error message?